flask/app.py

```diff
@@ -1769,8 +1769,13 @@ def handle_url_build_error(self, error, endpoint, values):
                 rv = handler(error, endpoint, values)
                 if rv is not None:
                     return rv
-            except BuildError as error:
-                pass
+            except BuildError as e:
+                """We can't directly save the BuildError in error because the python3 execution model changed.
```

Comment:
Does such a verbose comment even make sense here? It's kind of explaining the language itself which is not an appropriate use of comments. So I'd rather go for a short `# make error available outside except block (py3)` or something like that.